setup.cfg

```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = babi
-version = 0.0.25
+version = 0.0.26
```

Comment:
you should not bump versions in packages you don't own

Reply:
i just pulled the changes after the revert and created a new branch, i never changed the version manually.

Reply:
well whatever you did there was incorrect -- it shows as you comitting the version change: a75312affc66b44f0